### DISCUSSION TITLE
Mobile app hangs on gitkraken/vscode-gitlens repository

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#43967] [Mobile] [answered]
1. Open GitHub Mobile
2. Search for `gitkraken/vscode-gitlens`
3. Select the repository result

https://user-images.githubusercontent.com/2119212/211838349-7c7cef75-5cd3-461c-b282-c3eb00e36704.mp4

Sometimes it'll load the Readme content and then hang. It's quite painful.

Expected results:
* User should be able to add repository to favorites or go back to previous screen even if repository is heavy

**Comment by jacortinas** [comment, marked as answer]
Hi, @jsoref! Looking at that repository, the readme content does seem extremely heavy. I'll open up an issue with the team to see if there is anything we can do to make loading more robust. Thank you!

**Comment by stevepopovich** [comment]
Hi @jsoref, what version of the app are you using? What device and operating system version? We definitely have some issues loading large readmes sometimes, but this one seems to be working ok for me. Thanks!

**Reply by jsoref** [reply]
1.94 (Updated Jan 3, 2023) on Pixel 3A (Android 12)

I can opt into the beta.

**Reply by stevepopovich** [reply]
We would love for you to join the beta! https://play.google.com/apps/testing/com.github.android

Though I would expect similar performance in this case with the beta.

**Comment by stevepopovich** [comment]
Hi @jsoref is this still happening for you?

**Reply by jsoref** [reply]
Nope, it's fine now, thanks!
